tests/includes/handler/class-test-move.php

```diff
@@ -0,0 +1,257 @@
+<?php
+/**
+ * Test file for Activitypub Move Handler.
+ *
+ * @package Activitypub
+ */
+
+namespace Activitypub\Tests\Handler;
+
+use Activitypub\Collection\Followers;
+use Activitypub\Model\Follower;
+use Activitypub\Handler\Move;
+use Activitypub\Http;
+
+/**
+ * Test class for the Move handler.
+ */
+class Test_Move extends \WP_UnitTestCase {
+	/**
+	 * The user ID.
+	 *
+	 * @var int
+	 */
+	private $user_id;
+
+	/**
+	 * The second user ID.
+	 *
+	 * @var int
+	 */
+	private $user_id_2;
+
+	/**
+	 * Setup the test.
+	 */
+	public function setUp(): void {
+		parent::setUp();
+		$this->user_id   = $this->factory->user->create();
+		$this->user_id_2 = $this->factory->user->create();
+	}
+
+	/**
+	 * Tear down the test.
+	 */
+	public function tearDown(): void {
+		parent::tearDown();
+		wp_delete_user( $this->user_id );
+		wp_delete_user( $this->user_id_2 );
+	}
+
+	/**
+	 * Test the handle_move method with a target and origin.
+	 */
+	public function test_handle_move_with_target_and_origin() {
+		$target = 'https://example.com/new-profile';
+		$origin = 'https://example.com/old-profile';
+
+		// Mock the HTTP response for the origin object.
+		$origin_object = array(
+			'type'    => 'Person',
+			'id'      => $origin,
+			'url'     => $origin,
+			'name'    => 'Old Profile',
+			'inbox'   => 'https://example.com/old-profile/inbox',
+			'movedTo' => $target,
+		);
+
+		// Mock the HTTP response for the target object.
+		$target_object = array(
+			'type'          => 'Person',
+			'id'            => $target,
+			'url'           => $target,
+			'name'          => 'New Profile',
+			'inbox'         => 'https://example.com/new-profile/inbox',
+			'also_known_as' => array(
+				$origin,
+			),
+		);
+
+		// Create a follower for the origin.
+		$origin_follower = new Follower();
+		$origin_follower->set_inbox( 'https://example.com/old-profile/inbox' );
+		$origin_follower->set_name( 'Old Profile' );
+		$origin_follower->set_type( 'Person' );
+		$origin_follower->set_id( $origin );
+		$origin_follower->set_url( $origin );
+		$id = $origin_follower->upsert();
+
+		// Add the user ID meta value.
+		add_post_meta( $id, '_activitypub_user_id', $this->user_id );
+
+		// Mock the HTTP request.
+		add_filter(
+			'pre_http_request',
+			function ( $preempt, $args, $url ) use ( $target, $target_object, $origin, $origin_object ) {
```

Comment:
in `class-test-update.php` these `pre_http_request` callbacks get removed, let's do the same here